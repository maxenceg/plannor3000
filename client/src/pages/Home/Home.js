import * as React from 'react';

import Dashboard from '../../components/Dashboard';

export default class Home extends React.Component {
  render() {
    return (
      <div>
        <Dashboard />
      </div>
    );
  }
}
