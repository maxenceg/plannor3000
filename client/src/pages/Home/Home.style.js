const styles = {
  menuContainer: {
    padding: 15,
    fontSize: 14,
    fontFamily: "Roboto",
    color: "white",
  }
};

export default styles;
