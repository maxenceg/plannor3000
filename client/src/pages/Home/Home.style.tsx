import styled from 'styled-components';
import { Link } from 'react-router-dom';

const StyledIntro = styled.div`
  font-size: large;
`;

export const StyledLink = styled(Link)`
  text-decoration: none;
`;

export default StyledIntro;
