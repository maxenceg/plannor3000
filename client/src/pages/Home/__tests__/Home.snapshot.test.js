// @flow
import React from 'react';
import { shallow } from 'enzyme';

import Home from '../Home';

describe('<Home>', () => {
  it('should render correctly', () => {
    const tree = shallow(<Home />);
    expect(tree).toMatchSnapshot();
  });
});
