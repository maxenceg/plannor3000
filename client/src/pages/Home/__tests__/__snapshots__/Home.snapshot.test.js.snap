// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home> should render correctly 1`] = `
<Fragment>
  <Homestyle__StyledIntro>
    <WithStyles(Typography)
      gutterBottom={true}
      variant="subtitle1"
    >
      <FormattedMessage
        defaultMessage="To get started, edit"
        id="home.get-started"
        values={Object {}}
      />
    </WithStyles(Typography)>
    <WithStyles(Typography)
      variant="caption"
    >
      <code>
        src/App.js
      </code>
    </WithStyles(Typography)>
    <WithStyles(Typography)
      gutterBottom={true}
      variant="subtitle1"
    >
      <FormattedMessage
        id="home.save-to-reload"
        values={Object {}}
      />
    </WithStyles(Typography)>
    <WithStyles(Typography)
      variant="body1"
    >
      <FormattedMessage
        id="home.create-route"
        values={Object {}}
      />
    </WithStyles(Typography)>
    <WithStyles(Typography)
      variant="body1"
    >
      <FormattedMessage
        id="home.generate-component"
        values={Object {}}
      />
    </WithStyles(Typography)>
    <WithStyles(Typography)
      variant="caption"
    >
      <code>
        yarn generate
      </code>
    </WithStyles(Typography)>
    <WithStyles(Typography)
      variant="body1"
    >
      <FormattedMessage
        id="home.generate-module"
        values={Object {}}
      />
    </WithStyles(Typography)>
    <WithStyles(Typography)
      variant="body1"
    >
      <FormattedMessage
        id="home.readme"
        values={Object {}}
      />
    </WithStyles(Typography)>
    <Homestyle__StyledLink
      href="/avatar"
      to="/avatar"
    >
      <WithStyles(Button)
        color="primary"
        variant="contained"
      >
        <FormattedMessage
          id="home.use-a-link"
          values={Object {}}
        />
      </WithStyles(Button)>
    </Homestyle__StyledLink>
  </Homestyle__StyledIntro>
</Fragment>
`;
